docs/conf.py

```diff
@@ -66,6 +67,7 @@ def pick_newtype_attrs(some_type: type) -> list:
     "Rule": ("obj", "qrules.argument_handling.Rule"),
     "SpinFormalism": ("obj", "qrules.transition.SpinFormalism"),
     "StateDefinition": ("obj", "qrules.combinatorics.StateDefinition"),
+    "StateDefinitionInput": ("obj", "qrules.combinatorics.StateDefinition"),
```

Comment:
```suggestion
    "StateDefinitionInput": ("obj", "qrules.combinatorics.StateDefinitionInput"),
```

Reply:
Silly me, but I'm wondering why the docs still build without complaint.